templates/messaging-system/undesired-events/undesired-events.1.schema.json

```diff
@@ -0,0 +1,49 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "title": "undesired-events",
+  "properties": {
+    "impression_id": {
+      "description": "A UUID representing this event",
```

Comment:
The impression_id represents a client/user rather than an event, correct?

Reply:
Oops, that's correct. Let me correct all the descriptions for impression_id.